### ISSUE TITLE
New neovim error: context_commentstring nvim-treesitter module is deprecated

### ISSUE THREAD

**Issue by mikemee** [shaunsingh/nyoom.nvim#161]
Using nvim v0.9.5, when I start I get an error:

> context_commentstring nvim-treesitter module is deprecated

The details are here: https://github.com/JoosepAlviste/nvim-ts-context-commentstring/issues/82

The fix that worked for me was to delete this line in the file: `fnl/modules/tools/tree-sitter/config.fnl`

> :context_commentstring {:enable true}

Hope this helps someone else. Not sure how nyoom handles various versions of nvim, so I didn't submit a PR, but I'm happy to do so.

**Comment by phasewalk1** [comment]
yeah I was running into this issue on nvim 0.9.5 and ended up removing that same line as well
